File: src/dataset/go/darts_Exercism_go_drapala/openAI/ChatGPT4_darts_v1.go
package darts

func Score(x, y float64) int {
	r2 := x*x + y*y // Calculate squared radius to avoid expensive sqrt operation

	switch {
	case r2 <= 1.0:
		return 10 // Inner circle
	case r2 <= 25.0:
		return 5 // Middle circle
	case r2 <= 100.0:
		return 1 // Outer circle
	default:
		return 0 // Missed target
	}
}

File: src/dataset/Java/darts_exercism-java-uzilan_robiworks/openAI/ChatGPT4_darts_v1.java
class Darts {
    private final double distanceSquared;

    Darts(double x, double y) {
        this.distanceSquared = x * x + y * y;
    }

    int score() {
        if (distanceSquared <= 1) {
            return 10;
        } else if (distanceSquared <= 25) {
            return 5;
        } else if (distanceSquared <= 100) {
            return 1;
        } else {
            return 0;
        }
    }
}